website/index.html

```diff
@@ -510,6 +510,7 @@ <h2 id="linter-feature-comparison">Features</h2>
           <li><a href="demo/">try quick-lint-js in your browser</a></li>
           <li><a href="hiring.html">we're hiring! get paid to code</a></li>
           <li><a href="benchmarks/">benchmarks vs other linters</a></li>
+          <li><a href="cli.html">CLI documentation</a></li>
```

Comment:
Blocking: cli.html isn't deployed with the website, but should be. Add cli.html to `General Commands Manual`.